doc/api/cli.md

```diff
@@ -3543,8 +3543,14 @@ an increase of 1 MiB to semi-space applies to each of the three individual
 semi-spaces and causes the heap size to increase by 3 MiB. The throughput
 improvement depends on your workload (see [#42511][]).
 
-The default value is 16 MiB for 64-bit systems and 8 MiB for 32-bit systems. To
-get the best configuration for your application, you should try different
+In <=v19 the default value is 16 MiB for 64-bit systems and 8 MiB for 32-bit systems.
+In v20 and later, however, the default value depends on the memory limit
+and will be smaller than in previous versions for memory limits of 2 GiB or less.
+
+For example, a memory limit of 512 MiB will default to 1 MiB of semi space in v20,
+whereas in v18 the default for 64-bit systems was 16 MiB.
```

Comment:
This doc is only for v23, so there's no need to specify the behavior of older versions.